### DISCUSSION TITLE
Swap Keypad Behavior

### DISCUSSION THREAD

**Discussion by thiskappaisgrey** [meow-edit/meow#222] [Q&A] [answered]
Hi,
How do I swap the behavior of keypad? For example I want `SPC x t` to dispatch `Ctrl-x t` and not `Ctrl-x Ctrl-t`. Is there some way to customize this behavior? Thanks.

**Comment by eshrh** [comment, marked as answer]
Not right now. I think the behavior is pretty much hard coded in to some degree as well. I was able to get it functionally working by applying this [patch](https://gist.github.com/eshrh/c614d568835690eca12dbba7a7644f20). The biggest problem with this as far as i can tell is that the keymap description is wrong. You're welcome to try to hack on meow to fix it though! The relevant function is [here](https://github.com/meow-edit/meow/blob/master/meow-keypad.el#L138).

**Reply by thiskappaisgrey** [reply]
Thanks! Yeah, I'll look into it when I get the time.
